haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alex Hitech"
+LICENSE="MIT"
```

Comment:
Not only MIT, but also [BSD (3-clause)](https://github.com/HaikuArchives/Eventual/blob/1433b19ca24d7217ae104e5a7bc075979cbce701/src/PreferencesPreflet/License.txt)

Reply:
@alaviss Would I just type BSD? Or BSD (3-clause)?

Reply:
If I'm not mistaken, it should be
```
LICENSE="MIT
	BSD (3-clause)"
```
or maybe
```
LICENSE="BSD (3-clause)
	MIT"
```
I don't know if we sort them alphabetically or if we list those licenses which are mostly used in the source. So I guess you can chose whatever order you want.